apps/ledger-live-desktop/src/renderer/components/PostOnboardingHub/PostOnboardingMockAction.tsx

```diff
@@ -17,28 +21,34 @@ const PostOnboardingMockAction = ({ id }: Props) => {
   const history = useHistory();
   const action = getPostOnboardingAction(id);
 
+  const clearLastActionCompleted = useCallback(() => {
+    dispatch(clearPostOnboardingLastActionCompleted());
+  }, [dispatch]);
+
   const completeAction = useCallback(
     () => dispatch(setPostOnboardingActionCompleted({ actionId: id })),
     [dispatch, id],
   );
 
-  const navigateToDashboard = useCallback(() => {
-    setDrawer();
-  }, []);
-
-  const navigateToHub = useCallback(() => {
-    setDrawer(PostOnboardingHubContent);
-  }, []);
-
   const handleCompleteAndGoToDashboard = useCallback(() => {
     completeAction();
-    navigateToDashboard();
-  }, [completeAction, navigateToDashboard]);
+    setDrawer();
+    clearLastActionCompleted();
+  }, [clearLastActionCompleted, completeAction]);
 
   const handleCompleteAndGoToHub = useCallback(() => {
     completeAction();
-    navigateToHub();
-  }, [completeAction, navigateToHub]);
+    setDrawer(
```

Comment:
Maybe export this logic in an individual hook that you reuse everywhere else: like `useOpenPostOnboardingDrawerCallback`.